v2/internal/attacktechniques/aws/defense-evasion/cloudtrail-stop/main.tf

```diff
@@ -19,18 +19,18 @@ provider "aws" {
 }
 
 resource "aws_cloudtrail" "trail" {
-  name           = "my-cloudtrail-trail"
+  name           = "stratus-red-team-cloudtrail-stop-trail"
```

Comment:
for cloudtrail in particular, we can take it as an opportunity to add a random suffix to solve #147

Reply:
should we go ahead and add random suffixes to all of the resources? Just since we are already changing the resource names it seems to fit.

Reply:
let's maybe close this PR first and revisit, to make sure it doesn't stay open for too long and isn't too big?